### ISSUE TITLE
Search should only search for license key or license name

### ISSUE THREAD

**Issue by chinyeungli** [aboutcode-org/scancode-licensedb#16]
Searching 'mit' in the licensedb returns 307 licenses.
https://scancode-licensedb.aboutcode.org/?search=mit
It is because the search pick up the Copyleft Li"mit"ed in the Category field. However, I think the search should only apply to license_key/license_name, all other fields should not be in the query for the search.

**Comment by m-g-rathod** [comment]
Hello sir, here's my approach to this issue: 

Currently for querying the Datatable in **scancode-toolkit\src\licensedcode\templates\license_list.html** we have : 
**table.search().draw()** function in js which queries the entire datatable.

For applying search to only first 2 columns i.e. key and short_name columns we can use **columns() method** and pass the respective column names or indexes as argument in the function like:
**table.columns([0,1]).search().draw()**
OR
**table.columns(column_names or id).search().draw()**

I referred to this [Link](https://datatables.net/reference/api/columns().search()#Description) for solving this issue.

Please check and suggest whether my approach to this issue is correct or there's something more to do.

**Comment by AyanSinhaMahapatra** [comment]
@m-g-rathod yes that makes sense to me. See also https://github.com/nexB/scancode-licensedb/issues/30#issuecomment-1439773022 and follow the conversation there.
